flow/project.py

```diff
@@ -3359,14 +3337,14 @@ def _main_script(self, args):
 
         # Gather all pending operations or generate them based on a direct command...
         with self._potentially_buffered():
+            default_directives = self._get_default_directives()
             if args.cmd:
                 warnings.warn("The --cmd option for script is deprecated as of "
                               "0.9 and will be removed in 0.11.",
                               DeprecationWarning)
-                operations = self._generate_operations(args.cmd, jobs, args.requires)
+                operations = self._generate_operations(args.cmd, jobs, default_directives, args.requires)
```

Comment:
Here the functionality is to run a command provided on the command line. As such the default directives would have no directives for it. Also the default directives here is a `dict` of `dict`s (i.e a dictionary with operation name keys and operation directives, dictionaries, as values. Having this become the directives for a `JobOperation` does not make much sense. An empty `dict` would make more sense.